docs/quickstart.md

```diff
@@ -286,7 +286,7 @@ yarn install
 yarn start
 ```
 
-
+For more information, dive deeper into the [Unity Catalog UI](./ui.md).
```

Comment:
There is no `./ui.md` page per the lint test - could you please update to the valid location ?

Reply:
Please refer to [this check](https://github.com/unitycatalog/unitycatalog/actions/runs/11305261771/job/31444571561?pr=597) - thanks!

Reply:
Thank you for addressing these issues. I have corrected them and committed the changes. - thanks!